src/components/projects/ProjectCard.jsx

```diff
@@ -94,7 +94,10 @@ ProjectCard.propTypes = {
     title: PropTypes.string.isRequired,
     bodyText: PropTypes.string.isRequired,
     image: PropTypes.string,
-    links: PropTypes.arrayOf(PropTypes.object),
+    links: PropTypes.arrayOf(PropTypes.shape({
+      title: PropTypes.string.isRequired,
```

Comment:
the prop name is text and not title. please update it

Reply:
Thanks, I made another commit to update that.